lib/puppet/module.rb

```diff
@@ -351,6 +361,25 @@ def ==(other)
     self.environment == other.environment
   end
 
+  def initialize_i18n
+    locales_path = File.absolute_path('locales', path)
+    config_path = File.absolute_path('config.yaml', locales_path)
+    module_name = @forge_name.gsub("/","-") if @forge_name
+
+    nil if initialized_i18n? module_name
```

Comment:
Only at the ends of methods. This one is in the middle, and we _want_ to quit if this is true.